helpers/helpers_test.go

```diff
@@ -422,3 +422,31 @@ func TestParseOldCSR(t *testing.T) {
 		t.Fatalf("%v", err)
 	}
 }
+
```

Comment:
can I have some simple tests on the new behavior of LoadPEMCertPool? empty path and empty file.